forum/qa-plugin/user-activity-log/events-window/events-info-scripts.js

```diff
@@ -0,0 +1,15 @@
+const ModalWindow = ()=>{
```

Comment:
```suggestion
const setupModalWindow = () =>{
```
Nazwy pisane wielką literą są "zarezerwowane" dla klas